src/components/Metrics/components/ChartIncomingMessage.tsx

```diff
@@ -0,0 +1,152 @@
+import { Chart, ChartArea, ChartAxis, ChartGroup, ChartLegend, ChartThemeColor, ChartVoronoiContainer } from "@patternfly/react-charts";
```

Comment:
There should be no need to have a new chart component, the existing `ChartLinearWithOptionalLimit` should be enough to cover the need we have